packages/members-api/index.js

```diff
@@ -311,6 +310,7 @@ module.exports = function MembersApi({
         getMemberIdentityData,
         getPublicConfig,
         bus,
+        sendEmailWithMagicLink,
```

Comment:
This operation had to be separated from user creation and is exposed here so that caller can call it depending on the need instead of passing extra flags into model method or whatever.